src/components/my-app.js

```diff
@@ -183,52 +184,52 @@ class MyApp extends connect(store)(LitElement) {
-        <button class="menu-btn" title="Menu" on-click="${_ => store.dispatch(updateDrawerState(true))}">${menuIcon}</button>
+        <button class="menu-btn" title="Menu" @click="${_ => store.dispatch(updateDrawerState(true))}">${menuIcon}</button>
         <div main-title>${appTitle}</div>
       </app-toolbar>
 
       <!-- This gets hidden on a small screen-->
       <nav class="toolbar-list">
-        <a selected?="${_page === 'view1'}" href="/view1">View One</a>
-        <a selected?="${_page === 'view2'}" href="/view2">View Two</a>
-        <a selected?="${_page === 'view3'}" href="/view3">View Three</a>
+        <a ?selected="${_page === 'view1'}" href="/view1">View One</a>
+        <a ?selected="${_page === 'view2'}" href="/view2">View Two</a>
+        <a ?selected="${_page === 'view3'}" href="/view3">View Three</a>
       </nav>
     </app-header>
 
     <!-- Drawer content -->
-    <app-drawer opened="${_drawerOpened}"
-        on-opened-changed="${e => store.dispatch(updateDrawerState(e.target.opened))}">
+    <app-drawer .opened="${_drawerOpened}"
+        @opened-changed="${e => store.dispatch(updateDrawerState(e.target.opened))}">
       <nav class="drawer-list">
-        <a selected?="${_page === 'view1'}" href="/view1">View One</a>
-        <a selected?="${_page === 'view2'}" href="/view2">View Two</a>
-        <a selected?="${_page === 'view3'}" href="/view3">View Three</a>
+        <a ?selected="${_page === 'view1'}" href="/view1">View One</a>
+        <a ?selected="${_page === 'view2'}" href="/view2">View Two</a>
+        <a ?selected="${_page === 'view3'}" href="/view3">View Three</a>
       </nav>
     </app-drawer>
 
     <!-- Main content -->
     <main role="main" class="main-content">
-      <my-view1 class="page" active?="${_page === 'view1'}"></my-view1>
-      <my-view2 class="page" active?="${_page === 'view2'}"></my-view2>
-      <my-view3 class="page" active?="${_page === 'view3'}"></my-view3>
-      <my-view404 class="page" active?="${_page === 'view404'}"></my-view404>
+      <my-view1 class="page" ?active="${_page === 'view1'}"></my-view1>
+      <my-view2 class="page" ?active="${_page === 'view2'}"></my-view2>
+      <my-view3 class="page" ?active="${_page === 'view3'}"></my-view3>
+      <my-view404 class="page" ?active="${_page === 'view404'}"></my-view404>
     </main>
 
     <footer>
       <p>Made with &hearts; by the Polymer team.</p>
     </footer>
 
-    <snack-bar active?="${_snackbarOpened}">
+    <snack-bar ?active="${_snackbarOpened}">
         You are now ${_offline ? 'offline' : 'online'}.</snack-bar>
     `;
   }
 
   static get properties() {
     return {
-      appTitle: String,
-      _page: String,
-      _drawerOpened: Boolean,
-      _snackbarOpened: Boolean,
-      _offline: Boolean
+      appTitle: { type: String },
+      _page: { type: String },
+      _drawerOpened: { type: Boolean },
+      _snackbarOpened: { type: Boolean },
+      _offline: { type: Boolean }
```

Comment:
@sorvell If that's the case should lit-element just always treat `Boolean` as `BooleanAttribute`? Note also these properties are "private" so they wouldn't otherwise be set by attribute by an external element.